### PR TITLE
Add warnings on auth in the asset issuance guide.

### PR DESCRIPTION
**Pull request by theaeolianmachine**
* Added a warning around the usage of AUTHORIZATION REVOCABLE.
* Set using a single transaction as a best practice.

### CODE REVIEW

**Comment by theaeolianmachine** [comment]
Ping on this.

File: guides/issuing-assets.md [hunk 1]
```diff
@@ -252,26 +252,25 @@ server.submitTransaction(setHomeDomain);
...
+[`AUTHORIZATION REVOCABLE` flag](concepts/assets.md#controlling-asset-holders) allows you to freeze
+assets you issued in case of theft or other special circumstances. This can be useful for national
+currencies, but is not always applicable to other kinds of assets.
 
-If your asset is special purpose or you’d like to control who can be paid with it, use the [`AUTHORIZATION REQUIRED` flag](concepts/assets.md#controlling-asset-holders), which requires that the issuing account also approves a trustline before the receiving account is allowed to be paid with the asset.
+For most cases, you should avoid setting `AUTHORIZATION REVOCABLE` on your asset. In the past, some
+issuers have used the `AUTHORIZATION REVOCABLE` flag in order to impose lock-up periods. However,
+this is a problematic mechanism because it does not provide the user any guarantees with regard to
+when or if the assets will be unlocked.
```
Comment (tomerweller) [inline guides/issuing-assets.md]: There's another reason why it's problematic. It requires serious plumbing in order to undo.
Just turning the `AUTH_REQUIRED`/`AUTH_REVOCABLE` flags to `false` doesn't affect existing accounts. You'll need to manually `ALLOW_AUTH=true` on existing accounts which can only be done when the `AUTH_REQUIRED` flag is true. Which essentially means that, to do it correctly without affecting new accounts being created, you'll need to execute transactions that sandwich `ALLOW_AUTH=true` operations between setting `AUTH_REQUIRED=true` in the beginning and changing to `AUTH_REQUIRED=false` in the end.

**Comment by theaeolianmachine** [comment]
@tomerweller - I went ahead and gave this another pass based on what you mentioned. Let me know if anything's inaccurate in my interpretation of what you said; I'm happy to talk offline about it as well if anything's missing.

**Comment by theaeolianmachine** [comment]
@tomerweller and I reviewed in person, and I added a bit about sandwiching multiple allow trust operations as well.

**Pull request merged**